pkg/inject/xray_test.go

```diff
@@ -159,6 +192,70 @@ func Test_xrayMutator_mutate(t *testing.T) {
 									"cpu":    cpuRequests,
 									"memory": memoryRequests,
 								},
+								Limits: corev1.ResourceList{
```

Comment:
Would be good to add the same test case scenarios that are added for envoy around this.

Reply:
Added